envs/monkey_zoo/terraform/monkey_zoo.tf

```diff
@@ -309,6 +309,7 @@ resource "google_compute_instance_from_template" "mimikatz-15" {
     }
     auto_delete = true
   }
+  tags = ["test-machine","windows","windowsserver2016","mimikatz-15"]
```

Comment:
What are these tags?

Reply:
Network tags that we can use when we set firewall rules.

Reply:
Doesn't seem we need them. Nor have then on other machines

Reply:
Some of the firewalls depend on the network tags. I would rather use network tags than actual ips.